pkg/bootstrap/platform/gcp.go

```diff
@@ -98,6 +98,8 @@ var (
 		return fmt.Sprintf("https://container.googleapis.com/v1/projects/%s/locations/%s/clusters/%s",
 			projectID, clusterLocation, clusterName), nil
 	}
+
+	zonePattern = regexp.MustCompile("^[^-]+-[^-]+-[^-]+$")
```

Comment:
can you add a comment about what this is matching? possibly some examples?

Reply:
Done.